nexus/src/authz/omicron.polar

```diff
@@ -373,6 +373,10 @@ resource IpPoolList {
 has_relation(fleet: Fleet, "parent_fleet", ip_pool_list: IpPoolList)
 	if ip_pool_list.fleet = fleet;
 
+# Any authenticated user can create a child of a provided IP Pools.
+# This is necessary to use the pools when provisioning instances.
+has_permission(_actor: AuthenticatedActor, "create_child", _ip_pool: IpPool);
```

Comment:
I don't think it matters much right now since we don't really support fleets yet, but for consistency with other policy rules I think there should probably be an additional restriction here:
```
has_permission(actor: AuthenticatedActor, "create_child", ip_pool: IpPool)
        if actor.silo.fleet = ip_pool.fleet;
```

Reply:
Done

Reply:
Hrm, this appears to actually break the parsing of the oso file:

```
    error_message_internal: failed to compute authorization: trace (most recent evaluation last):
      009: allow(oso::host::value::PolarValue TYPE `oso::host::value::PolarValue`, oso::host::value::PolarValue TYPE `oso::host::value::PolarValue`, oso::host::value::PolarValue TYPE `oso::host::value::PolarValue`)
      008: actor.authenticated and
        has_permission(actor.authn_actor.unwrap(), action.to_perm(), resource)
        in rule allow at line 21, column 5
      007: actor.authn_actor
        in rule allow at line 22, column 20
      006: actor.authn_actor.unwrap()
        in rule allow at line 22, column 20
      005: action.to_perm()
        in rule allow at line 22, column 48
      004: has_permission(actor.authn_actor.unwrap(), action.to_perm(), resource)
        in rule allow at line 22, column 5
      003: actor.silo.fleet = ip_pool.fleet
        in rule has_permission at line 379, column 6
      002: actor.silo.fleet = ip_pool.fleet
        in rule has_permission at line 379, column 6
      001: actor.silo.fleet
        in rule has_permission at line 379, column 6
     
    Application error: Attribute fleet not found on type Option. at line 379, column 6:
        379:   if actor.silo.fleet = ip_pool.fleet;
```

Reply:
I'm unwrapping in a11fc2689fd8c7c262c64710c4fd712751183f91 , though I'm open to a safer way to do this in polar.